docs/JSON-RPC-API/Authentication.md

```diff
@@ -0,0 +1,88 @@
+# Authentication and Authorization 
+
+Authentication identifies a user based on a username and password. Authorization verifies whether the user has
+access to the JSON-RPC method they are requesting.  
+
+Pantheon uses [JWT tokens](https://jwt.io/introduction/) to authenticate JSON-RPC API requests. 
```

Comment:
We use JWT tokens to authorize requests. For authentication, we use username/password.

Reply:
Fixed